README.md

```diff
@@ -1,78 +1,120 @@
+  </tr>
+</table>
+
+# What's supported in DetectionMetrics
+
+<table><thead>
+  <tr>
+    <th>Task</th>
+    <th>Modality</th>
+    <th>Datasets</th>
+    <th>Framework</th>
+  </tr></thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Segmentation</td>
+    <td>Image</td>
+    <td>Rellis3D, GOOSE, custom GAIA format</td>
+    <td>PyTorch, Tensorflow</td>
+  </tr>
+  <tr>
+    <td>LiDAR</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td>Object detection</td>
+    <td>Image</td>
+    <td>Check <a href="/v1"><i>DetectionMetrics v1</i></a></td>
+    <td>Check <a href="/v1"><i>DetectionMetrics v1</i></a></td>
+  </tr>
+</tbody>
+</table>
+
+More details about the specific metrics and input/output formats required fow each framework are provided in the [Compatibility](/v2/compatibility) section in our webpage.
 
 
 # Installation
+In the near future, *DetectionMetrics* is planned to be deployed in PyPI. In the meantime, you can clone our repo and build the package locally using either *venv* or *Poetry*.
 
-### Install packaged image
-
-To quickly get started with Detection Metrics, we provide a docker image.
-
-* Download docker image and run it
+### Using venv
+Create your virtual environment:
 ```
-    docker run -dit --name detection-metrics -v [local_directory]:/root/volume/ -e DISPLAY=host.docker.internal:0 jderobot/detection-metrics:noetic
+mkdir .venv
+python3 -m venv .venv
 ```
 
-This will start the GUI, provide a configuration file (appConfig.yml can be used) and you are ready to go. Check out the [web](https://jderobot.github.io/DetectionMetrics) for more information
-
-### Installation from source (Linux only)
-
-Check the installation guide [here](https://jderobot.github.io/DetectionMetrics/installation/). This is also the recommended installation 
-for **contributors**.
-
-# Starting with Detection Metrics
-Check out the [beginner's tutorial](https://jderobot.github.io/DetectionMetrics/resources/tutorial/).
-
-# General Detection Metrics GUI
-
-The top toolbar shows the different tools available.
-
-<p align="center">
-  <img  style="border: 5px solid black;" width="800" height="500" src="docs/assets/images/main_window.png" />
-</p>
-
+Activate your environment and install as pip package:
+```
+source .venv/bin/activate
+pip install -e .
+```
 
-# Example of detection and console output in Detection Metrics
+### Using Poetry
 
-Two image views are displayed, one with the ground truth and the other with the detected annotations.
-In the console output, log info is shown.
+Install Poetry (if not done before):
+```
+python3 -m pip install --user pipx
+pipx install poetry
+```
 
-![detector](docs/assets/images/detector.png)
+Install dependencies and activate poetry environment (you can get out of the Poetry shell by running `exit`):
+```
+poetry install
+poetry shell
+```
 
+### Common
+Install your deep learning framework of preference in your environment. We have tested:
```

Comment:
Is the application multiplatform?

Reply:
in the sense of different OSs?

Reply:
yep, it was an issue with the first version due to ROS constraints but not anymore as I can see!